srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -119,6 +126,19 @@
 	    			"aTargets": [0]
 	    		}]
 	    	});
+        	// Get a DataTables API instance
+        	var apiTable = proceduresTable.dataTable().api();
+        	// Unbind the default global search and keyup
+        	$("div.dataTables_filter input").unbind();
+        	//Add a "starts with" regex to the global search
+        	// Enable regex search and disable smart searching
+        	$("div.dataTables_filter input").on('keyup', function () {
+        		apiTable.column(1).search('^' + this.value, true, false).draw();
+        	});
+        	// Use the CPT code for the proper radio button and set it to checked
+        	$("input:radio[value='${variableEntry.dynamicValues['Procedure']}']").prop("checked",true);
```

Comment:
This will break if the procedure variable is ever not called "Procedure". I recommend moving this line inside the variable foreach above so you can access that actual variable key.